src/reactviews/pages/TableExplorer/TableExplorerToolbar.tsx

```diff
@@ -36,9 +38,8 @@ export const TableExplorerToolbar: React.FC<TableExplorerToolbarProps> = ({
         context.createRow();
     };
 
-    // Use cell-level change count directly
-    // This provides accurate granularity: each cell edit counts as one change
-    const changeCount = cellChangeCount;
+    // Total changes includes both cell edits and row deletions
+    const changeCount = cellChangeCount + deletionCount;
```

Comment:
the correctness of this sum doesn't matter, just whether it's 0, right?  otherwise, I'd wonder how it handles editing a cell then deleting the row it's in, but I don't think that matters.